content/en/docs/concepts/security/pod-security-admission.md

```diff
@@ -136,6 +139,35 @@ current policy level:
 - Valid updates to `.spec.activeDeadlineSeconds`
 - Valid updates to `.spec.tolerations`
 
+## Pod Security Webhook
```

Comment:
this is already in the 1.23 branch at https://github.com/kubernetes/website/blob/dev-1.23/content/en/docs/concepts/security/pod-security-admission.md?plain=1#L47